.github/workflows/test-frontend-unit.yml

```diff
@@ -0,0 +1,54 @@
+name: "Frontend test suite"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - dev
+      - release/*
+    paths:
+      - '**/frontend/**/*.ts'
+      - '**/frontend/**/*.js'
+      - '**/frontend/**/*.json'
```

Comment:
It would be great if someone could check these filter patterns. I was caught out at first by GH Actions lack of support for extended globbing (see [[Filter pattern cheat sheet](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)).